src/Zobrist.h

```diff
@@ -28,11 +28,13 @@
 
 class Zobrist {
 public:
+    static constexpr auto zobrist_empty = 0x1234567887654321ULL;
```

Comment:
Not 100% sure but I think as long as there's an 0x prefix then the ULL part is not actually required.

Reply:
fixed